reflex/components/radix/themes/layout/list.py

```diff
@@ -42,29 +42,32 @@ class BaseList(Component):
     tag = "ul"
 
     # The style of the list. Default to "none".
-    list_style_type: Var[
+    list_style_type: Optional[
```

Comment:
```suggestion
    list_style_type: Var[
```

This one can also be a `Var`, since the value just gets shoved into the style dict